backend/app/Resources/views/print/index.html.twig

```diff
@@ -70,15 +70,14 @@
     <div class="content">
```

Comment:
I think that the changes in this file should not be here. @fiddike Your opinion is welcome.